spring-messaging/src/main/java/org/springframework/messaging/handler/invocation/reactive/HandlerMethodReturnValueHandler.java

```diff
@@ -33,7 +33,10 @@ public interface HandlerMethodReturnValueHandler {
 
 	/** Header containing a DataBufferFactory for use in return value handling. */
 	String DATA_BUFFER_FACTORY_HEADER = "dataBufferFactory";
-
+	/**
+	 * Message header name of response content type.
+	 **/
+	String RESPONSE_CONTENT_TYPE = "responseContentType";
```

Comment:
It's not clear what the purpose of this header is. It seems to be set as an empty AtomicReference from `MessagingRSocket` in the beginning, and then its value is set from `AbstractEncoderMethodReturnValueHandler`, but I don't see anything making use of it.